pennylane/debugging.py

```diff
@@ -298,10 +505,45 @@ def _measure(measurement):
 
 
 def tape():
-    """Access the quantum tape of the circuit.
+    """Access the tape of the quantum circuit.
+
+    The tape can then be drawn to visualize the gates that have
+    been applied from the quantum circuit so far.
```

Comment:
Doesn't need to be exactly this, but something like:
```suggestion
    The tape can then be used to access all properties stored in :class:`~pennylane.tape.QuantumTape`.
    This can be used to visualize the gates that have
    been applied from the quantum circuit so far or otherwise process the operations.
```